### ISSUE TITLE
miner runs but gives me only booo's

### ISSUE THREAD

**Issue by blackdiamont** [Max-Coin/cpuminer#5]
Hi,
miner runs fine, but i only got boo's :/

[2014-03-02 23:35:55] Starting Stratum on stratum+tcp://us-east1.maxcoin.mining

poolhub.com:10461

[2014-03-02 23:35:55] 2 miner threads started, using 'keccak' algorithm.

[2014-03-02 23:35:55] Binding thread 0 to cpu 0

[2014-03-02 23:35:55] Binding thread 1 to cpu 1

[2014-03-02 23:36:04] Stratum detected new block

[2014-03-02 23:36:07] thread 0: 1596198 hashes, 58.29 khash/s

[2014-03-02 23:36:38] thread 1: 1824114 hashes, 58.07 khash/s

[2014-03-02 23:36:38] accepted: 0/1 (0.00%), 114.71 khash/s (booooo)

Tried to get it running on my synology ds213+ nas.
Under Windows its working with the same args.
any ideas?

**Comment by zone117x** [comment]
Yeah this code is totally bugged. It submits invalid low diff shares. From what I've seen the maxcoin devs have borked everything they have touched. Cudaminer submits valid shares though.

**Comment by lukem512** [comment]
Can you confirm that the miner downloaded from http://rocketpool.co.uk/max/ works with your set up?

**Comment by zone117x** [comment]
Nope. That cpuminer is also bugged, it submits low-diff invalid shares. All the other miners work though.

**Comment by lukem512** [comment]
zone, what version of Stratum are you running? This is the first we've heard of such issues. Can you give us a complete breakdown of what you're doing so we can try and reproduce this, please.

**Comment by zone117x** [comment]
I'm using [node-stratum-pool](https://github.com/zone117x/node-stratum-pool) which so far supports everything but maxcoin and its forks/clones because of this issue. 

I'm simply trying to add support for maxcoin and other cloned coins which have the same changes with the block creation process. In the other issue in the maxcoin repo I've given a very detailed explanation of everything I'd tried, links to my code, theories on why its not working, etc.
